cmd/vicadmin/logs.go

```diff
@@ -378,14 +378,15 @@ func findSeekPos(f *os.File) int64 {
 		// Go back nBytes from the last newline we've seen (or beginning-of-file),
 		// and read the next nBytes
 		readPos := lastnl - int64(len(buf))
-		if pos < 0 {
+		// Fix here.  Previously was "pos < 0".
+		if readPos < 0 {
```

Comment:
This could easily be the issue as Go source makes it very unclear what the behaviour would be with a negative position. Looks like it gets passed unmodified to syscall.pread.